client/landing/stepper/declarative-flow/internals/steps-repository/importer-migrate-message/style.scss

```diff
@@ -2,9 +2,9 @@
 @import "@automattic/onboarding/styles/mixins";
 
 .import-hosted-site,
+.hosted-site-migration-v2 :not(:has(.list__wrapper)),
```

Comment:
`:not(:has(.list__wrapper))`

It prevents a bug on the platform-identification screen. There is a `svg` selector that is messing the color icons. 
We probably should fix it.